docs/usage/auto-updates.md

```diff
@@ -55,6 +55,24 @@ For further documentation on configurations, check the [updates strategy][update
 [strategy-fleet_lock]: updates-strategy.md#lock-based-strategy
 [updates-strategy]: updates-strategy.md
 
+## Updates ordering and downgrades
+
+OS updates have a strict ascending ordering called "age index", which is based on the date and time of release.
+Versions that have been released earlier in time have a lower index than recent ones.
+
+Zincati uses this absolute ordering to prefer newer releases (i.e. with higher age index) when multiple updates are available at the same time.
+By default, this ordering is also used to prevent automatic downgrades.
+
+For custom environments where automatic downgrades have to be supported, the following configuration snippet can be used to enable them:
+
+```toml
+[updates]
+allow_downgrade = true
+```
+
+Enabling such logic removes an additional safety check which may be valuable in case of rogue Cincinnati servers.
```

Comment:
This line reads a bit off to me. I believe it's meant to note that removing the logic would allow a rogue/malicious Cincinnati server to provide compromised or broken updates. What about:

```
[...] may allow rogue Cincinnati servers to provide broken or malicious updates.
```

or something like that?